lib/browser/utils.ts

```diff
@@ -18,7 +18,7 @@ export function createLazyInstance (
 ) {
   let lazyModule: Object
   const module: any = {}
-  for (const method in (holder as any).prototype) {
+  for (const method in (holder as any).prototype) { // eslint-disable-line guard-for-in
```

Comment:
hmm is this correct? we should probably guard with `hasOwnProperty` here...

Reply:
We want to lazily access inherited prototypes here, so I think `in` is right 🤔